packages/augur-ui/src/modules/modal/add-funds.tsx

```diff
@@ -135,7 +152,25 @@ export const AddFunds = ({
           <BackButton action={() => setSelectedOption(() => null)} />
           <CloseButton action={() => closeAction()} />
         </div>
-        <div className={selectedOption === '3' ? Styles.AddFundsTransfer : Styles.AddFundsCreditDebitCoinbase}>
+        <div
+          className={
+            selectedOption === '3'
```

Comment:
Can you use constants for ‘0’ and ‘3’ to make it more readable

Reply:
👍